dummy-data.js

```diff
@@ -1,92 +1,92 @@
 const dummyData = [
   {
```

Comment:
This file also needs to be moved onto the same level as your index.js file in order for it to work. React doesn't compile or use anything outside of the `src` folder.